CredentialProvider.Microsoft.Tests/CredentialProviders/Vsts/AuthUtilTests.cs

```diff
@@ -48,29 +47,26 @@ public void TestCleanup()
         [TestMethod]
         public async Task GetAadAuthorityUri_WithoutAuthenticateHeaders_ReturnsCorrectAuthority()
```

Comment:
>GetAadAuthorityUri_WithoutAuthenticateHeaders_ReturnsCorrectAuthority [](http://example.com/codeflow?start=26&length=69)

checking if there were already MSAL versions of these and these are now duplicates and can be removed